modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm

```diff
@@ -56,17 +56,15 @@
 	ears = /obj/item/radio/headset/headset_medsec
 	uniform = /obj/item/clothing/under/rank/security/peacekeeper/security_medic
 	gloves = /obj/item/clothing/gloves/color/latex/nitrile
-	shoes = /obj/item/clothing/shoes/combat/peacekeeper
+	shoes = /obj/item/clothing/shoes/jackboots
 	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
 	suit = /obj/item/clothing/suit/armor/vest/peacekeeper/security_medic
 	l_hand = /obj/item/storage/firstaid/brute
 	head = /obj/item/clothing/head/beret/sec/peacekeeper/security_medic
 
-	backpack_contents = list(/obj/item/melee/baton/telescopic, /obj/item/storage/box/gunset/security_medic)
-
-	backpack = /obj/item/storage/backpack/security/peacekeeper
-	satchel = /obj/item/storage/backpack/satchel/sec/peacekeeper
-	duffelbag = /obj/item/storage/backpack/duffelbag/sec/peacekeeper
+	backpack = /obj/item/storage/backpack/security
+	satchel = /obj/item/storage/backpack/satchel/sec
+	duffelbag = /obj/item/storage/backpack/duffelbag/sec
```

Comment:
You did this before, but I didn't get around to mentioning it, there's no real reason for this. BlueSec should really be the default, as it's a Skyrat thing. Right now, we have an issue where there's a mix-match of security running around, and it just looks bad.

Reply:
You can alt-click your equipment to reskin it how you wish.

Reply:
I know that much. My main problem is how RedSec is now the default. Instead of Bluesec, for like, no reason.

Reply:
the reason is bluesec sucks

Reply:
just because something is a "skyrat thing" doesn't mean its a good thing